src/utils.js

```diff
@@ -21,6 +21,11 @@ export function getNodeName(node) {
 }
 
 export function getEventName(attrName) {
+  const isSpecialCaptureEvent = attrName.substr(-7) === 'Capture' && attrName.substr(-14, 7) !== 'Pointer';
```

Comment:
This logic we should put on updateAttribute method to find `isCaptureEvent` event. This case handles both the normal capture case and special case. So it's not just the `isSpecialCaptureEvent `

And we can pass isCaptureEvent as param in this function.